selfapi.js

```diff
@@ -367,6 +372,21 @@ API.prototype = {
     });
   },
 
+  // Build index routes.
+  toAPIIndex: function (basePath) {
```

Comment:
Actually, I copied the code from `toHTML` as I was expecting this method to behave the same. What do you think?